components/jira/actions/list-issue-comments/list-issue-comments.mjs

```diff
@@ -1,33 +1,34 @@
-// legacy_hash_id: a_NqilRo
-import { axios } from "@pipedream/platform";
+import jira from "../../jira.app.mjs";
 
 export default {
   key: "jira-list-issue-comments",
   name: "List Issue Comments",
-  description: "Lists all comments for an issue.",
-  version: "0.1.1",
+  description: "Lists all comments for an issue, [See the docs](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-comments/#api-rest-api-3-issue-issueidorkey-comment-get)",
+  version: "0.1.2",
   type: "action",
   props: {
-    jira: {
-      type: "app",
-      app: "jira",
-    },
+    jira,
     issueIdOrKey: {
-      type: "string",
-      description: "The ID or key of the issue to get comments of.",
+      propDefinition: [
+        jira,
+        "issueIdOrKey",
+      ],
     },
     startAt: {
       type: "integer",
+      label: "Start at",
       description: "The index of the first item to return in a page of results (page offset).\nDefault: `0`, Format: `int64`.",
       optional: true,
     },
```

Comment:
This prop `startAt` can be removed. We should handle any pagination ourselves either in the component or in the app file. I like to handle pagination in the app file, and then calling `this.jira.listIssueComments` would return a list of paginated results.

Reply:
makes sense thank you